server/eventlog/eventlog_test.go

```diff
@@ -0,0 +1,105 @@
+package eventlog_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/buildbuddy-io/buildbuddy/server/backends/chunkstore"
+	"github.com/buildbuddy-io/buildbuddy/server/eventlog"
+	"github.com/buildbuddy-io/buildbuddy/server/interfaces"
+	"github.com/buildbuddy-io/buildbuddy/server/tables"
+	"github.com/buildbuddy-io/buildbuddy/server/testutil/mockstore"
+	"github.com/buildbuddy-io/buildbuddy/server/testutil/testenv"
+	"github.com/buildbuddy-io/buildbuddy/server/util/status"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	aclpb "github.com/buildbuddy-io/buildbuddy/proto/acl"
+	elpb "github.com/buildbuddy-io/buildbuddy/proto/eventlog"
+	telpb "github.com/buildbuddy-io/buildbuddy/proto/telemetry"
+)
+
+type MockInvocationDB struct {
```

Comment:
I know there's not a lot here, but you could put this in server/testutil/mockinvocationdb if you wanted to make it reusable